src/SFA.DAS.EmployerFinance.UnitTests/Events/ProcessDeclarationTests/WhenIProcessDeclarations.cs

```diff
@@ -49,5 +58,98 @@ public async Task ThenTheLoggerIsCalledWithInfoLevel()
             //Assert
             _logger.Verify(x=>x.Info("Process Declarations Called"));
         }
+
+        [Test]
+        public async Task ThenLevyAddedToAccountEventIsPublishedForGreaterThanZeroLevyAmount()
+        {
+            long accountId = 900034;
+            decimal levyAmount = 10.0m;
+
+            _dasLevyRepository
```

Comment:
Can the duplicated code from the 3 tests be refactored out so we don't have 3 copies?

Reply:
Yes it can; I sort of thought about that as I was copying and pasting, but went with the lazy option :)